src/main/java/carracing/domain/car/Car.java

```diff
@@ -38,4 +33,11 @@ public String getCarName() {
         return carName;
     }
 
+    public boolean isMaxPosition(int maxPosition) {
+        return maxPosition <= position;
+    }
+
+    public Object clone() throws CloneNotSupportedException{
+        return super.clone();
+    }
```

Comment:
```suggestion
    public Car clone() {
        try {
            return (Car) super.clone();
        } catch (CloneNotSupportedException e) {
            throw new RuntimeException(e.getMessage());
        }
    }
```
개인적으로 특별한 이유가 없으면 Exception을 부모 메소드에서 처리하기보단 호출하는 곳에서 처리하는것을 권장드려요 !